docs/platform-integration/device-media/text-to-speech.md

```diff
@@ -11,6 +11,34 @@ This article describes how you can use the .NET Multi-platform App UI (.NET MAUI
 
 The default implementation of the `ITextToSpeech` interface is available through the `TextToSpeech.Default` property. Both the `ITextToSpeech` interface and `TextToSpeech` class are contained in the `Microsoft.Maui.Media` namespace.
 
+## Get started
+
+To access text-to-speech functionality, the following platform-specific setup is required.
+
+# [Android](#tab/android)
+
+If your project's Target Android version is set to **Android 11 (R API 30)** or higher, you must update your _Android Manifest_ with queries that use Android's [package visibility requirements](https://developer.android.com/preview/privacy/package-visibility).
```

Comment:
This URL redirects to https://developer.android.com/about/versions/11/privacy/package-visibility. But that page is also relatively useless and invites you to navigate to https://developer.android.com/training/package-visibility. So maybe that's the URL to link to.

Reply:
Also think it's better to say something like "....you must update your _Android Manifest_ with an intent filter for the text-to-speech (TTS) engine...".

Reply:
```suggestion
If your project's Target Android version is set to **Android 11 (R API 30)** or higher, you must update your _Android Manifest_ with an intent filter for the text-to-speech (TTS) engine. For more information about intents, see Android's documentation on [Intents and Intent Filters](https://developer.android.com/guide/components/intents-filters).
```